### PR TITLE
Error messages for create declaration definition

### LINKED ISSUE #10163: Create declaration/definition should display error message
Create declaration or definition feature should display an error message of an operation that doesn't complete.

### PR DESCRIPTION
**Pull request by michelleangela**
Add error messages for create declaration definition feature.

### CODE REVIEW

**Review by sean-mcmanus — DISMISSED** [review]

File: Extension/src/nativeStrings.json [hunk 1]
```diff
@@ -368,5 +368,74 @@
     },
     "cm_addfunction": {
         "text": "Automatic add function"
-    }
+    },
+    "e_com_virtual_redundant": {
+        "text": "vsCMFunctionVirtual is redundant and must not be specified when with vsCMFunctionComMethod.",
+        "hint": "Do not localized 'vsCMFunctionVirtual' and 'vsCMFunctionComMethod', they are code implementation."
```
Comment (sean-mcmanus) [inline Extension/src/nativeStrings.json:374]: localized->localize

(multiple spots)

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]

File: Extension/src/nativeStrings.json [hunk 2]
```diff
@@ -368,5 +368,74 @@
...
+        "hint": "%s is the file that has an invalid path."
+    },
+    "e_cm_file_not_in_project": {
+        "text":  "File '%s' was not found.",
+        "hint": "%s is the file that was not found."
+    },
+    "refactor_create_declaration_definition_failed":
+    {
+        "text": "Create Declaration / Definition failed:",
+        "hint": "The operation 'Create Declaration / Definition' on a function was not successfully."
```
Comment (sean-mcmanus) [inline Extension/src/nativeStrings.json:407]: successfully->successful.

Comment (michelleangela) [reply Extension/src/nativeStrings.json:407]: I just checked-in. I'll fix in next PR. I have other strings to fix.

Comment (sean-mcmanus) [reply Extension/src/nativeStrings.json:407]: ok

**Pull request merged**